changelogs/unreleased/4388-fix-deprecation-warning-setuptools.yml

```
@@ -0,0 +1,4 @@
+description: "fix deprecation warning for setuptools"
+change-type: patch
+destination-branches: [iso5, master]
```

```suggestion
destination-branches: [iso5]
```
If I understand correctly. It's been merged in master already, right?

---

no It has not, It only succeeded for iso4. I will change the PR message